x-pack/legacy/plugins/apm/public/components/app/Settings/AddSettings/AddSettingFlyoutBody.tsx

```diff
@@ -155,6 +181,42 @@ export function AddSettingFlyoutBody({ onSubmit }: { onSubmit: () => void }) {
           />
         </EuiFormRow>
 
+        {selectedConfig ? (
+          <>
+            <EuiHorizontalRule margin="m" />
+            <EuiTitle size="xs">
+              <h3>
+                <EuiText color="danger">Delete configuration</EuiText>
+              </h3>
+            </EuiTitle>
+
+            <EuiSpacer size="s" />
+
+            <EuiText>
+              <p>
+                If you wish to delete this configuration, please be aware that
+                the agents will continue to use the existing configuration until
+                it syncs with the APM Server.
+              </p>
+            </EuiText>
+
+            <EuiSpacer size="s" />
+
+            <EuiButton
+              fill={false}
+              color="danger"
+              onClick={async () => {
+                await deleteConfig(selectedConfig.id);
+                onSubmit();
+              }}
+            >
+              Delete
+            </EuiButton>
+
+            <EuiSpacer size="m" />
+          </>
+        ) : null}
+
         <EuiFormRow>
           {/* TODO: Move to AddSettingFlyout EuiFlyoutFooter instead */}
           <EuiButton
```

Comment:
I guess this is replaced by the submit button in the `EuiFlyoutFooter` in `AddSettingFlyout`?